src/components/TokenTotalLineItem.test.tsx

```diff
@@ -74,7 +74,9 @@ describe('TokenTotalLineItem', () => {
     it('shows the right amounts', () => {
       const { getByTestId } = renderComponent({})
       expect(getElementText(getByTestId('TotalLineItem/Total'))).toEqual('R$15.00')
-      expect(getElementText(getByTestId('TotalLineItem/ExchangeRate'))).toEqual('R$1.50')
+      expect(getElementText(getByTestId('TotalLineItem/ExchangeRate'))).toEqual(
+        'tokenExchanteRate, {"symbol":"cUSD"}R$1.50'
```

Comment:
These look a bit weird, but that's fine.

Reply:
i agree, this one is weird. but the `tokenExchanteRate` translation is `"{{symbol}} @ <0></0>"` which is the source of the weirdness i think....i didn't want to investigate why for this 🙈